index.html

```diff
@@ -0,0 +1,15 @@
+<!--
```

Comment:
Interesting. How was it able to work before? 

Is it made available for the Cypress Vite dev server? I saw that it has a vite config file too.

Would make sense if this was the case since Vite will look for an `index.html` file to use in the root dir directory.

Reply:
Before, the `vite.config` file looks in `playground` for the root `index.html` and watches the directories relative to that file. Our `vite.config` file only has simple defaults.